packages/x-scheduler/src/joy/event-calendar/EventCalendar.tsx

```diff
@@ -12,9 +12,21 @@ export const EventCalendar = React.forwardRef(function EventCalendar(
   const { events, onEventsChange, className, ...other } = props;
 
   return (
-    <div className="mui light">
-      <div className={clsx(className, 'EventCalendar')} ref={forwardedRef} {...other}>
-        HELLO WORLD ({events.length} events loaded)
+    <div
+      className={clsx(className, 'EventCalendarRoot', 'mui', 'light')}
+      ref={forwardedRef}
+      {...other}
+    >
+      <div className="EventCalendarSidePanel">
+        <span style={{ height: 72, paddingTop: 18 }}>TODO: Time nav</span>
```

Comment:
I used inline style here because those elements are not meant to stay for long.
Unlike the Calendar placeholder that we will probably not have anytime soon.